pkg/network/event_common.go

```diff
@@ -274,8 +276,11 @@ type ConnectionStats struct {
 
 	IntraHost bool
 	IsAssured bool
+	IsClosed  bool
 
-	ContainerID *string
+	ContainerID struct {
+		Source, Dest *intern.Value
```

Comment:
you can use `*intern.StringValue` here from `pkg/util/intern` to reduce allocations. this unboxes to interface. also you can remove the cast when getting this value (thanks bryce for showing me this)

Reply:
nevermind, discussed in a thread on `#system-probe`